test-clients/src/main/resource/spec-default.properties

```diff
@@ -89,10 +89,14 @@ node.details.name=NODE_DETAILS
 # Valid settings are { fixed, random }
 node.selector=fixed
 nodes=localhost
+# custom testnet
+# nodes=3.13.199.218
+# custom mainnet
+# nodes=54.162.70.169
 # nightly regression
-#nodes=52.45.177.185:0.0.3,3.213.224.169:0.0.4,54.196.133.58:0.0.5,3.214.13.242:0.0.6
+# nodes=52.45.177.185:0.0.3,3.213.224.169:0.0.4,54.196.133.58:0.0.5,3.214.13.242:0.0.6
```

Comment:
space introduced. No need for this right. same issue two lines below

Reply:
Ok, fixed.